concepts/functions/about.md

```diff
@@ -1,3 +1,302 @@
 # About
 
-TODO: add information on functions concept
+A [`function`][function] is a block of organized, reusable code that is used to perform a single, related action.
+`Functions` provide better modularity for your application and a high degree of code reusing.
+Like other programming languages, python provides _pre-defined functions_ ([`print`][print], [`map`][map] etc) that are readily available in the Python language. 
```

Comment:
These have a technical name: built-in functions. It would be good to use the proper name when they are first introduced. 
It might be good to link to them, too: `https://docs.python.org/3/library/functions.html`

Reply:
This line also has both `python` and `Python`. Both are used throughout this file.